packages/tizen_rpc_port/README.md

```diff
@@ -0,0 +1,59 @@
+# tizen_rpc_port
+
+[![pub package](https://img.shields.io/pub/v/tizen_rpc_port.svg)](https://pub.dev/packages/tizen_rpc_port)
+
+Tizen applications can communicate with each other using RPC Port APIs, which allows applications to send and receive data after a connection is established.
+
+>**Note**
+>
+> APIs will be used in generated source files by TIDLC which is an Interface Definition Language (IDL) compiler for Tizen.
+> **[TIDL](https://docs.tizen.org/application/native/guides/app-management/tidl/) is recommended rather than using RPC Port APIs directly.**
```

Comment:
A detailed description on how to generate Dart code using TIDL (and also how to use the API) should be added to the README.

Reply:
We will add usage for dart in TIDL docs.
Do you want to put it in this README as well?

Reply:
No, that will be enough I guess.

Reply:
We will modify it more clearly in the future.
Then, how about adding an example of using tidlc to this?
```
Usage example:
  stub : tidlc -s -l dart -i Message.tidl -o MessageStub
  proxy: tidlc -p -l dart -i Message.tidl -o MessageProxy
```

Reply:
That would be nice. (I'll reformat and clean up the markdown document in the future.)

Also if you want to use terms such as "proxy" and "stub" instead of "client" and "server", please make sure to describe their meaning clearly in the linked document for those who don't have enough background knowledge.

Reply:
Ok, we will consider that part when modifying it.
Thank you very much!